### ISSUE TITLE
[request] Add "Pause" button/functionality (if possible)

### ISSUE THREAD

**Issue by HarikalarKutusu** [alicevision/meshroom#1444]
**Is your feature request related to a problem? Please describe.**
In smaller projects, this is not so important but in my last project, I really needed this. One node (no chunks) can take multiple hours (in my case whole day). During this process, I continue to work on other things (daily work). But because the computer is overhauled (i7-8700K / 32GB RAM), I had to cancel the process and lost many hours of work several times.

**Describe the solution you'd like**
I know, this can not be solved very easily as Meshroom uses external processes and has to work on multiple OS's. Also the memory it uses cannot be freed, be it RAM or GPU. That would also mean a special dump/save state to disk functionality (but I'm not good at OS-level programming).

At least "pause after the current function is finished" can be a simpler solution.

**Describe alternatives you've considered**
I currently go to the project and cancel the following nodes' execution. But at least from the UX point of view, a pause button would mean a lot.

**Comment by natowi** [comment]
> pause after the current function is finished

I think you mean "pause after current chunk is completed",  yes this would be neat.

However even now you don´t loose all progress when stopping Meshroom, as long as you did not remove or override the cache folder, all nodes and chunks that have been processed up to this point will be reused when you resume the computation.

**Comment by HarikalarKutusu** [comment]
Exactly, I meant chunks... And indeed, I did your method at first, open Meshroom on one of my monitors, watch the logs scroll, wait for the completion, then I press stop, save & close. But, as some chunks can be as long as 3 hours this turned out to be tiresome, so I found cancelling follow-up nodes to be a better solution.
